.buildkite/premerge.steps.yaml

```diff
@@ -76,19 +76,25 @@ steps:
     <<: *windows
     artifact_paths:
       - logs/**/*
+    env:
+      ACCELERATOR_ENDPOINT: "unity-accelerator.production-intinf-eu1.i8e.io"
```

Comment:
Can these be set on queue level? So we set it once per windows/mac ?

Reply:
We could probably set this in the `pre-command` hook, make use of `BUILDKITE_AGENT_META_DATA_PLATFORM` that the agent will know

Reply:
I feel like you can set the env for a given agent queue.

cc @petemounce ?

Reply:
Moved it up, going to test and see if this works

Reply:
You can set it, but it'll be wiped if you define env per step:

see the environment tab on the `build` steps here: https://buildkite.com/improbable/gdk-for-unity-premerge/builds/4019